test/test_erase.py

```diff
@@ -87,6 +86,38 @@ def test_erase_with_target_id(self):
         self.assertEqual(ret, 0)
         self.assertEqual(len(devices), len(mbeds.list_mbeds()))
 
+    # test func name is larger than 30, but is meaningful
+    # pylint: disable=invalid-name
+    def test_erase_failed_non_supported_devices(self):
+        mbeds = mbed_lstools.create()
+        devices = mbeds.list_mbeds(read_details_txt=True)
+        devices = [dev for dev in devices if dev.get('daplink_automation_allowed') == '0']
+        count = len(devices)
+        if count == 0:
+            self.skipTest('no suitable test')
```

Comment:
This is workaround for https://github.com/ARMmbed/mbed-ls/issues/288

Reply:
Sorry about that, we'll work on getting a patch up for that soon

Reply:
this is not big problem - at least in this case :)